docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Lightning (Coming soon)
+sidebar_label: Lightning
+---
+
+## The Lightning pitch
```

Comment:
Can we please have a different title than `The Lightning pitch`? This sounds like it's copied from internal draft documents ... maybe `Introducing Lightning`